backend/LexBoxApi/Controllers/ProjectController.cs

```diff
@@ -232,6 +232,17 @@ private async Task StreamHttpResponse(HttpContent hgResult)
         await hgResult.CopyToAsync(writer.AsStream());
     }
 
+    [HttpGet("getLdmlZip")] // TODO: Discuss endpoint name, and whether it should be GET or POST, at next opportunity
```

Comment:
I think a GET is fine since it's not actually modifying data. maybe `sldr-export`?

Reply:
Done in commit 72f1a52e.